src/PJV.ts

```diff
@@ -502,8 +502,7 @@ const validateUrlTypes = (
 	return errors;
 };
 
-// TODO: Should we consider deprecating the PJV named export in favor of directly
-// exporting the functions and constants?
+/** @deprecated please use the individual validate function */
```

Comment:
[Docs] Nit: using a `@link` gives a nice intellisense clicky:

```suggestion
/** @deprecated please use the individual @{link validate} function */
```

Reply:
Updated